pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -626,9 +626,9 @@ func selectVictimsOnNode(
 	// inter-pod affinity to one or more victims, but we have decided not to
 	// support this case for performance reasons. Having affinity to lower
 	// priority pods is not a recommended configuration anyway.
-	if fits, _, err := core.PodPassesFiltersOnNode(ctx, ph, state, pod, nodeInfo); !fits {
-		if err != nil {
-			klog.Warningf("Encountered error while selecting victims on node %v: %v", nodeInfo.Node().Name, err)
+	if status := fh.RunFilterPluginsWithNominatedPods(ctx, state, pod, nodeInfo); !status.IsSuccess() {
+		if status.AsError() != nil {
```

Comment:
we need to check the error code here, asError turns any non success status into an error, which is not what we want.

Reply:
+1, `if status.Code() == framework.Error`